Jenkinsfile

```diff
@@ -156,6 +156,13 @@ def Tasks = [
         ++$scala testingExample$v/testHtmlJSDom  &&
     sbtretry ++$scala testSuiteJVM$v/test testSuiteExJVM$v/test &&
     sbtretry ++$scala testSuite$v/test &&
+    sbtretry ++$scala \
+        testSuite$v/saveForStabilityTest \
+        testSuite$v/checkStability \
+        testSuite$v/forceRelinkForStabilityTest \
+        testSuite$v/checkStability \
+        testSuite$v/clean \
+        testSuite$v/checkStability &&
```

Comment:
Have you considered using a scripted test for this?

Seems like that might be simpler: https://www.scala-sbt.org/1.x/docs/Testing-sbt-plugins.html#step+4%3A+write+a+script

Reply:
I could use scripted, but it's harder to make it generic wrt. the `scalaVersion`.

Also, this is easier to reproduce by hand for iterative development.

Reply:
Perhaps more importantly: in a scripted test I would not have access to the entire `test-suite`. And I want to test with the entire test suite because that's the only thing that contains "a bit of everything". In a scripted tests, I'd have to manually reproduce a codebase causing all kinds of sources of instability, which would defeat the purpose.